log/log.go

```diff
@@ -79,11 +79,11 @@ func Warn(ctx context.Context, event string, opts ...option) {
 	eventFuncInst.f(ctx, event, WARN, opts...)
 }
 
-func ErrorDetails(ctx context.Context, event string, opts ...option) {
+func ErrorDetails(ctx context.Context, event string, err error, opts ...option) {
```

Comment:
Just want to check, but did we ever get a steer on whether we want this to be called `ErrorDetails` or `Error`, and the rename the existing `Error` function to something else?